nodes/test/nodes_tests/test_plugins.py

```diff
@@ -186,7 +190,7 @@ def tearDown(self):
         shutil.rmtree(self.childfs)
         Consumer.get_collection().remove()
         Bind.get_collection().remove()
-        Repo.get_collection().remove()
+        model.Repository.drop_collection()
```

Comment:
not sure it makes a big difference, remove clears the contents of a collection but the collection remains. drop_collection deletes the collection as well as the contents. I noticed the change in behavior and thus was curious as to why.